routes/api/contacts-router.js

```diff
@@ -0,0 +1,92 @@
+import express from "express";
+
+import contactsService from "../../models/contacts.js";
+
+import { HttpError } from "../../helpers/index.js";
+
+import Joi from "joi";
+
+const contactsRouter = express.Router();
+
+const contactAddSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
+  phone: Joi.string().required(),
+});
+
+contactsRouter.get("/", async (req, res, next) => {
+  try {
+    const result = await contactsService.listContacts();
```

Comment:
functions in routes should be in separate file. THey must just be invoked here

Reply:
Done